src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,5 @@
 NuGet.Commands.VerifyArgs.PackagePaths.get -> System.Collections.Generic.IReadOnlyList<string>
 NuGet.Commands.VerifyArgs.PackagePaths.set -> void
-static NuGet.Commands.MSBuildProjectFrameworkUtility.GetProjectFramework(string projectFilePath, string targetFrameworkMoniker, string targetPlatformMoniker, string targetPlatformMinVersion) -> NuGet.Frameworks.NuGetFramework
\ No newline at end of file
+NuGet.Commands.PackArgs.AliasMappings.get -> System.Collections.Generic.IDictionary<string, string>
+NuGet.Commands.PackArgs.AliasMappings.set -> void
```

Comment:
To someone using our API with no context of this issue, I'm not sure that `AliasMappings` is clear. Mapping to/from what?

Maybe `AliasFolderNameMapping` would be a bit clearer.